packages/core/src/components/navigation/Navbar/Navbar.tsx

```diff
@@ -51,43 +58,45 @@ function Navbar() {
               </Link>
             </>
           )}
+
           <SearchInput />
-          <div
-            data-fs-navbar-buttons
-            data-fs-navbar-search-expanded={searchExpanded}
-          >
+
+          <UINavbarButtons data-fs-navbar-search-expanded={searchExpanded}>
```

Comment:
Same here

Reply:
Good thought, but I didn't put it inside the component because it's external logic applied to the component, you know? It's not like he depends on that logic to exist, but we can discuss that with the team